### ISSUE TITLE
[FR] Safari support

### ISSUE THREAD

**Issue by purrloftruth** [shinigami-eyes/shinigami-eyes#16]
Would be great if there was a Safari version of this extension, so I could use it with my primary browser.

**Comment by lis0r** [comment]
Not very easy with the limitations on Safari plugins.

An alternative might be a version that's a greasemonkey userscript. I've used this method with AdGuard to adapt masstagger to Safari in the past, with variable success.

**Comment by purrloftruth** [comment]
The new version of Safari releasing this fall, Safari 14, will support WebExtensions:

https://developer.apple.com/documentation/safariservices/safari_web_extensions

**Comment by zorae** [comment]
It is trivial to make Shinigami Eyes work in Safari using Xcode's built-in WebExtension converter:

1. Install XCode
2. Run `sudo xcode-select -s /Applications/Xcode.app`
3. Install Shinigami Eyes in Chrome
4. Run `xcrun safari-web-extension-converter ~/Library/Application Support/Google/Chrome/Default/Extensions/ijcpiojgefnkmcadacmacogglhjdjphj`
5. In Xcode, select the "Shinigami Eyes (macOS)" scheme
6. Run the selected scheme
7. In Safari, choose "Develop > Allow Unsigned Extensions"
8. Enable the Shinigami Eyes extension in Safari preferences

As far as I can tell, all of Shinigami Eyes' features work in Safari. Accounts are colored correctly, I can use the right-click menu to tag accounts, and I can change the color scheme from the settings window.

**Comment by butterscotch69** [comment]
> It is trivial to make Shinigami Eyes work in Safari using Xcode's built-in WebExtension converter:
> 
> 1. Install XCode
> 2. Run `sudo xcode-select -s /Applications/Xcode.app`
> 3. Install Shinigami Eyes in Chrome
> 4. Run `xcrun safari-web-extension-converter ~/Library/Application Support/Google/Chrome/Default/Extensions/ijcpiojgefnkmcadacmacogglhjdjphj`
> 5. In Xcode, select the "Shinigami Eyes (macOS)" scheme
> 6. Run the selected scheme
> 7. In Safari, choose "Develop > Allow Unsigned Extensions"
> 8. Enable the Shinigami Eyes extension in Safari preferences
> 
> As far as I can tell, all of Shinigami Eyes' features work in Safari. Accounts are colored correctly, I can use the right-click menu to tag accounts, and I can change the color scheme from the settings window.

I tried this but it says 
Errors for "Shinigami Eyes":
The service_worker script failed to load due to an error.

I can go into the settings for the extension but it doesn't seem to work outside of that.

**Comment by sdaitzman** [comment]
Same issue here as @butterscotch69. Commenting to watch if someone else comes up with a solution before I get around to it.

**Comment by smiba** [comment]
Seems to work just fine for me following @zorae 's instructions 

Using:
Device: Macbook Pro 16" (2019) (x86)
OS: MacOS 13.3.1
SafarI 16.4 
XCode 14.3

**Comment by smiba** [comment]
@sdaitzman Yes, for example if I go to twitter I can see people's names change colour
